spyderlib/widgets/arraybuilder.py

```diff
@@ -256,7 +260,12 @@ def keyPressEvent(self, event):
             QDialog.keyPressEvent(self, event)
 
     def event(self, event):
-        if event.type() == QEvent.KeyPress and event.key() == Qt.Key_Tab:
+        """
+        Qt Override
+
+        Usefull when in line edit mode.
+        """
+        if event.type() == QEvent.KeyPress and event.key() == Qt.Key_Tab:  # pragma: nocover

```

Comment:
Maybe the easiest way is to call directly `event()` with a custom event.
Anyway you shouldn't hide the fact that it's not tested, even if it lowers the coverage stat. Actual measurement is better than a better score.